promise/model/LastPromiseModel.js

```diff
@@ -9,11 +9,13 @@ import {observable, action, computed} from '@xh/hoist/mobx';
 /**
  * Tracks the resolution state of the last of a series of promise invocations.
  * Currently it only tracks the *last* invocation, but could be enhanced to maintain historical statistics.
+ * Has an observable message property that if set will be displayed in a linked loadMask
```

Comment:
I'd move this to a doc comment right on the property itself.

Also - do we want a constructor now to set this initially?